react-facet/docs/api/mount-components.md

```diff
@@ -28,11 +28,11 @@ const Example = () => {
 
 ## `Map`
 
-Mounts a list of components based on a Facet of an array.
+Mounts a list of components based on a facet of an array.
```

Comment:
I noticed that there are inconsistencies with the capitalization of the word "facet." Is the correct spelling _Facet_ or _facet_? I presumed it's the latter, but please let me know if this was an incorrect presumption and I will change the capitalization.

Reply:
We don't have a rule, but I think lower case might be better, otherwise it becomes too much. Let's do that! Thanks